openshift/sso74-x509.yaml

```diff
@@ -269,7 +287,7 @@ objects:
     creationTimestamp: null
     name: ${NAME}${SUFFIX}
   spec:
-    minAvailable: 2
+    maxUnavailable: 1
```

Comment:
always allow 2 pods?

Reply:
setting maxUnavailable = 1 so that we are requiring at least 2 pods to be available at any certain time.